apps/condo/domains/banking/schema/BankIntegrationAccessRight.test.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Generated by `createschema banking.BankIntegrationAccessRight 'integration:Relationship:BankIntegration:PROTECT; user:Relationship:User:CASCADE;'`
+ */
+
+const { makeLoggedInAdminClient, makeClient } = require('@open-condo/keystone/test.utils')
+const {
+    expectToThrowAuthenticationErrorToObj, expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj, expectToThrowAccessDeniedErrorToObjects,
+} = require('@open-condo/keystone/test.utils')
+
+const { BANK_INTEGRATION_IDS } = require('@condo/domains/banking/constants')
+const { BankIntegrationAccessRight, createTestBankIntegrationAccessRight, updateTestBankIntegrationAccessRight, BankIntegration } = require('@condo/domains/banking/utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser, makeClientWithSupportUser, makeClientWithServiceUser } = require('@condo/domains/user/utils/testSchema')
+
+describe('BankIntegrationAccessRight', () => {
+    let adminClient, serviceUserClient, integration
+    beforeAll(async () => {
+        adminClient = await makeLoggedInAdminClient()
+        serviceUserClient = await makeClientWithServiceUser()
+        integration = await BankIntegration.getOne(adminClient, { id: BANK_INTEGRATION_IDS.SBBOL })
+    })
+
+    it('user: create BankIntegrationAccessRight', async () => {
+        const client = await makeClientWithNewRegisteredAndLoggedInUser()
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await createTestBankIntegrationAccessRight(client, integration, serviceUserClient.user)
+        })
+    })
+
+    it('anonymous: create BankIntegrationAccessRight', async () => {
+        const client = await makeClient()
+        await expectToThrowAuthenticationErrorToObj(async () => {
+            await createTestBankIntegrationAccessRight(client, integration, serviceUserClient.user)
+        })
+    })
+
+    it('support: create BankIntegrationAccessRight', async () => {
+        const support = await makeClientWithSupportUser()
+        const [integrationAccessRight] = await createTestBankIntegrationAccessRight(support, integration,  serviceUserClient.user)
+        expect(integrationAccessRight).toEqual(expect.objectContaining({
+            integration: { id: integration.id },
```

Comment:
Could you please retain standard checks from codegen

```
expect(obj.dv).toEqual(1)
expect(obj.sender).toEqual(attrs.sender)
expect(obj.v).toEqual(1)
expect(obj.newId).toEqual(null)
expect(obj.deletedAt).toEqual(null)
expect(obj.createdBy).toEqual(expect.objectContaining({ id: admin.user.id }))
expect(obj.updatedBy).toEqual(expect.objectContaining({ id: admin.user.id }))
expect(obj.createdAt).toMatch(DATETIME_RE)
expect(obj.updatedAt).toMatch(DATETIME_RE)
```

Reply:
Ok, I'll do it in another PR